res/database/Default/database_ads.xml

```diff
@@ -148,7 +148,20 @@ Naprawdę trzeba żuć.</pl>
 			<conditions min_audience="4.17" min_image="0" target_group="0" pro_pressure_groups="0" contra_pressure_groups="0" />
 			<data quality="15" repetitions="4" duration="3" profit="537" penalty="900" infomercial_profit="53" fix_infomercial_profit="1" />
 		</ad>
-		<ad guid="329eb726-34c8-49b3-a0af-be76dcbd296c" comment="TODO: availability years + predecessor">
+		<ad guid="b41a3658-c32c-4e80-8078-7ed09a359e82">
+			<title>
+				<all>Sticker Fuzzy</all>
+			</title>
+			<description>
+				<de>Räm Bäm Bäääm - Knall dir jetzt die Sticker deiner Lieblingsstars in unser Sammelheft.</de>
+				<en>Ramm Bam Baaam - Pop the stickers of your favorite stars into our scrapbook now.</en>
+				<pl>Ram tam taaam - Umieść naklejki z ulubionymi gwiazdami w naszym scrapbooku już teraz.</pl>
+			</description>
+			<conditions min_audience="0.39" min_image="4" target_group="2" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data quality="10" repetitions="2" duration="3" profit="2000" penalty="3220" fix_infomercial_profit="1" />
+			<availability year_range_to="2000" />
```

Comment:
Availability ranges are "exclusive"
from=2000
to=2001

means: 
any year _smaller_ than 2000 is excluded
any year _bigger_ than 2001 is excluded
-> 2000 _and_ 2001 are allowed.

In your case this means:
Sticker Fuzzy is available from X till (including) 1999.
Ringtone Fuzzy is available from 2002 till X.

Reply:
are you sure? it's wrong everywhere then.
even before "my time" it was done that way (e.g. "PoliPlay")

Reply:
Note that the return value for isAvailable is False in the snippet. So actually, the years defined in the database are inclusive. (If currentYear >2000 Then not available.)